src/tool/GenerateAction.cpp

```diff
@@ -10,10 +10,10 @@
 //
 
 #include "ToolArgs.hpp"
-#include "Lib/AbsoluteCompilationDatabase.hpp"
-#include "Lib/ConfigImpl.hpp"
-#include "Lib/CorpusImpl.hpp"
-#include "Lib/ToolExecutor.hpp"
+#include <lib/Lib/AbsoluteCompilationDatabase.hpp>
+#include <lib/Lib/ConfigImpl.hpp>
+#include <lib/Lib/CorpusImpl.hpp>
+#include <lib/Lib/ToolExecutor.hpp>
```

Comment:
I would use quotes for private includes but its just a cosmetic thing...

Reply:
OK. I'll undo that.

I usually use quotes when it's private and it's in the same directory because of how quotes invert the search procedure.